Makefile

```diff
@@ -101,19 +101,22 @@ $(JAVATOOLSBUILDDIR)/classes/%.class: tools/java/src/%.java
 		-sourcepath tools/java/src -d $(JAVATOOLSBUILDDIR)/classes $<
 
 # Build the Chisel emulator
+#emulator: OLD
+#	-mkdir -p $(HWBUILDDIR)
+#	$(MAKE) -C hardware BOOTBUILDROOT=$(CURDIR) BOOTBUILDDIR=$(BUILDDIR) BOOTAPP=$(BOOTAPP) BOOTBIN=$(BUILDDIR)/$(BOOTAPP).bin BOARD=$(BOARD) emulator
+#	-mkdir -p $(HWINSTALLDIR)/bin
+#	cp $(HWBUILDDIR)/emulator $(HWINSTALLDIR)/bin/patemu
+
+# chisel3/verilator emulator
+CORECNTT:=$(shell lscpu | grep 'Core(s) per socket:')
 emulator:
```

Comment:
Why is the emulator target gone?

Reply:
I kept the old target the new emulator target is below